.github/workflows/recreate-database.yml

```
@@ -0,0 +1,38 @@
+name: Re-create the database
+
+on: 
+  schedule:
+    - cron: "0 0 * * 1"
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: print secrets
+      - run: 'echo "$CERTIFICATE_KEY" > crt.key'
+        shell: bash
+        env:
+          SSH_KEY: ${{secrets.CERTIFICATE_KEY}}
+      - run: 'echo "$CERTIFICATE_PEM" > crt.pem'
+        shell: bash
+        env:
+          SSH_KEY: ${{secrets.CERTIFICATE_PEM}}
```

I think you meant this?
```suggestion
          CERTIFICATE_PEM: ${{secrets.CERTIFICATE_PEM}}
```